History.txt

```diff
@@ -1,11 +1,18 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
-Changes after FreeRTOS 202012.00 release
+Changes after FreeRTOS 202012.00-LTS and 202012.01-LTS releases

```

Comment:
Do you mean changes from FreeRTOS 202012.00-LTS to FreeRTOS 202012.01-LTS (this release)?

Reply:
Fixed